src/libstore/daemon.cc

```diff
@@ -970,6 +970,14 @@ static void performOp(TunnelLogger * logger, ref<Store> store,
         break;
     }
 
+    case wopCheckTrustedUser: {
+        logger->startWork();
+        bool result = trusted && store->checkTrustedUser();
```

Comment:
```suggestion
    case wopIsTrustedUser: {
        logger->startWork();
        bool result = trusted && store->isTrustedUser();
```

These two symbols are only about retrieving the information, but not necessarily acting upon it, so it's not really a check.
The `nix store check-trusted` command does not need renaming, because returning an exit code is more of a side effect.

Reply:
I've kept the local function name `checkTrustedUser()` in check-trusted.cc and doctor.cc, but otherwise each store implementation to `isTrustedUser`. Thanks for teaching me more about naming conventions!